lib/repl.js

```diff
@@ -1559,3 +1589,30 @@ function Recoverable(err) {
 }
 inherits(Recoverable, SyntaxError);
 exports.Recoverable = Recoverable;
+
+
+function parseInputParameteres(params) {
+  let started = false;
+  return params.split(',').reduce((acc, cur, i, array) => {
+     if(/'|"|`|\/\*|\*\//.test(cur)) {
+        if(started === true) {
+          acc[acc.length - 1] = acc[acc.length - 1] + "," + cur;
+          started = false;
+        } else {
+           acc.push(cur);
+           started = true;
+        }
+     } else {
+        acc.push(cur)
+     }
+     return acc;
+  }, []);
+}
+
+// add these to tests
+// console.log("::a, b::", annotate("a, b") )
+// console.log("::a, b = 0, /*c,*/ d::", annotate("a, b = 0, /*c,*/ d"))
+// console.log("::a, \"bamieh, ahm\", \"incom::", annotate("a, \"bamieh, ahm\", \"incom"))
+
+exports.replComplete = Symbol('REPL Complete');
```

Comment:
i feel like this name is a bit unclear, maybe `tabComplete` and `REPL Tab Complete`?

Reply:
thanks for the review! As i said the code still needs some work, but i wanted to put it out ASAP to get feedback in order to know if i should put in more time into this feature